bin/gen-api-package

```diff
@@ -195,6 +195,17 @@ var parseArgs = function parseArgs() {
     }
   );
   cli.addArgument(
+    [ '--experimental_alt_java' ],
+    {
+      defaultValue: false,
+      action: 'storeTrue',
+      help: 'When set indicates that the alternate build is used for java.\n'
+            + ' This makes that the generated gradle files will build packages '
```

Comment:
nit: s/packages '/packages'/  (remove the trailing whitespace), because the next line starts with a whitespace as well.